src/components/TodoList.jsx

```diff
@@ -1,64 +1,158 @@
 import React from 'react';
 import TodoPoint from './TodoPoint';
+import formatDate from './utils';
 
 const todoStyle = {
     display: "inline-block"
 }
-
-
+// Sort users by several categories
+const filterTasks = (sortingOpt) => {
```

Comment:
Let's extract this to utils.js, component files are better off being clean and short.